packages/teams-js/src/public/tasks.ts

```diff
@@ -37,8 +37,7 @@ export namespace tasks {
    */
   export function startTask(taskInfo: TaskInfo, submitHandler?: startTaskSubmitHandlerFunctionType): IAppWindow {
     const dialogSubmitHandler = submitHandler
-      ? /* eslint-disable-next-line strict-null-checks/all */ /* fix tracked by 5730662 */
-        (sdkResponse: dialog.ISdkResponse) => submitHandler(sdkResponse.err, sdkResponse.result)
+      ? (sdkResponse: dialog.ISdkResponse) => submitHandler(sdkResponse.err ?? '', sdkResponse.result ?? '')
```

Comment:
>submitHandler [](http://example.com/codeflow?start=46&length=13)

The two parameters to the `startTaskSubmitHandlerFunctionType` do not accept `undefined` values. However, the two properties on `sdkResponse` could be `undefined`. There's no good way to fix this without redesigning the API (e.g., the signature of `startTaskSubmitHandlerFunctionType` cannot be changed to accepted `undefined` because that would be a breaking API change). Consequently, to only pass legal values to the submit handler, `undefined` will get translated to the allowable empty string value.